### PR TITLE
Add React Person

### PR DESCRIPTION
**Pull request by ruslanpashkov**
[DEMO LINK](https://ruslanpashkov.github.io/react_person/)

### CODE REVIEW

**Review by DidenkoDenis306 — APPROVED** [review]

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,37 @@
...
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const partner = sex === 'm'
```
Comment (silchyn) [inline src/components/Person/Person.jsx:12]:
```suggestion
  } = person;
  
  const partner = sex === 'm'
```

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,37 @@
...
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried ? (
+          `${partnerName} is my ${partner}`
+        ) : (
+          'I am not married'
+        )}
```
Comment (silchyn) [inline src/components/Person/Person.jsx:33]:
```suggestion
        {isMarried
          ? (
            `${partnerName} is my ${partner}`
          ) : (
            'I am not married'
          )}
```

**Review by silchyn — CHANGES_REQUESTED** [review]
Read checklist carefully!

**Review by dinaminahi — APPROVED** [review]
Well done 🥇

**Pull request closed without merging**